core/src/test/kotlin/fr/sncf/osrd/conflicts/IncrementalConflictDetectorTests.kt

```diff
@@ -0,0 +1,63 @@
+package fr.sncf.osrd.conflicts
+
+import fr.sncf.osrd.standalone_sim.result.ResultTrain
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class IncrementalConflictDetectorTests {
+    data class SimpleRequirement(
+        val id: Int,
```

Comment:
Nit, a little bit clearer to me:
```suggestion
        val zone_id: Int,
```

Reply:
Fixed in https://github.com/OpenRailAssociation/osrd/pull/9980/commits/d9a0f84e16ff9c84831a2165a9cf23f8c4676afb